os_migrate/roles/import_workloads/defaults/main.yml

```diff
@@ -4,5 +4,6 @@ os_migrate_conversion_keypair_private_path: "{{ os_migrate_data_dir }}/conversio
 os_migrate_timeout: 1800
 os_migrate_workload_cleanup_on_failure: true
 os_migrate_workload_boot_volume_prefix: os-migrate-
+os_migrate_workload_stop_before_migration: false
```

Comment:
This should be documented

Reply:
About `os_migrate_workload_stop_before_migration` do we want to only allow stopping VMs before the migration? if we do this we will be changing the operator "regular behavior", do we want to allow them to start those stopped VMs after?

When stopping the VMs is useful? When the VMs have state, i.e. a DB server in that case is useful to stop/migrate but then the apps will fails as the DB server is stopped.

Something like `os_migrate_workload_start_after_migration` I think will be a nice addition, we could have only the "stop-only" and a follow up PR to handle this logic.

Reply:
+1 for docs, but for now i'd be -1 on `start_after_migration`. I think that once the workload is migrated, we want it running in the destination cloud, rather than running it duplicately? There might be some special cases where duplicating is the desired case but IMO let's discuss that separately + create a separate task for that if we indeed want it.

Reply:
Docs added in https://github.com/os-migrate/os-migrate/pull/464/commits/2a310a2820827fe8651e9867d6e2eb2c7d5c8859